frontend/src/components/Plan/Plan.module.css

```diff
@@ -0,0 +1,100 @@
+.topContainer {
+  display: grid;
+  padding: 0;
+  grid-template-columns: 0.5fr repeat(5, 1fr);
+  margin: 1rem;
+  margin-right: 30px;
+}
+
+ul {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.planBtn {
+  background: none;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+.burgerBtn {
+  width: 0;
+}
+
+.burgerBtnWrapper {
+  display: flex;
+  justify-content: start;
+}
+
+.collectedContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 19rem;
+  height: 5.5rem;
+  flex-flow: wrap;
+  background-color: #F5f5f7;
+  border-radius: 10px;
+  box-shadow: 5px 5px 15px #d7d7d7;
+  grid-column: 7;
+  padding: 0.5rem;
+  gap: 0.5rem;
+}
+
+.specialPersonWrapper {
+  display: flex;
+  text-align: left;
+  grid-column: 2/5;
+  height: 100%;
+  align-items: center;
+}
+
+.specialPerson {
+  margin: 0;
+}
+
+#collectedH4 {
```

Comment:
does this need to be an id?

Reply:
generally speaking, your default behavior is typically to use classes. you're not wrong in that from a technical pov, id's are used to target one HTML element but generally the more important philosophical question is whether or not you should be writing styling that is highly specific to one element or rather a class (or component). there is a whole argument about specificity as well where, for better or worse, ID's overrule all other CSS and can cause some unintended behaviors

Reply:
Oh good to know, I didn't know that ID's overrule all other CSS